docs/content/preview/yugabyte-platform/manage-deployments/upgrade-software-install.md

```diff
@@ -83,7 +83,7 @@ Currently, you cannot downgrade a universe to an older YugabyteDB release. For a
 
 1. If you are performing a rolling upgrade, specify the delay between node upgrades.
 
-    The delay allows the newly restarted node to stabilize before proceeding to the next node. This is a heuristic that varies according to the workload profile of the application hitting the database.
+    The delay allows the newly restarted node to stabilize before proceeding to the next node. This value implies the minimal amount of time allocated for all the operations with the node including, stop and start. This is a heuristic that varies according to the workload profile of the application hitting the database.
```

Comment:
```suggestion
    The delay allows the newly restarted node to stabilize before proceeding to the next node. For example, it may take some time for the overall query performance to stabilize as leader roles move away and back to this node because read caches take time to warm up. This value implies the minimal amount of time allocated for all the operations with the node including, stop and start. This is a heuristic that varies according to the workload profile of the application hitting the database.
```